PWGHF/TableProducer/treeCreatorLcToPKPi.cxx

```diff
@@ -519,7 +519,7 @@ struct HfTreeCreatorLcToPKPi {
                            float FunctionY,
                            float FunctionE) {
         double pseudoRndm = trackPos1.pt() * 1000. - (int64_t)(trackPos1.pt() * 1000);
-        if (FunctionSelection >= 1 && pseudoRndm < downSampleBkgFactor && candidate.pt() < downSampleBkgPtMax) {
+        if (FunctionSelection >= 1 && (candidate.pt() > downSampleBkgPtMax || (pseudoRndm < downSampleBkgFactor && candidate.pt() < downSampleBkgPtMax))) {
```

Comment:
Shouldn't the ` && candidate.pt() < downSampleBkgPtMax` part be dropped as it is trivially `true` inside that condition?